addons/website/static/tests/tours/snippet_popup_and_animations.js

```diff
@@ -0,0 +1,119 @@
+/** @odoo-module */
+
+import wTourUtils from "website.tour_utils";
+
+const snippets = [
+    {
+        id: "s_popup",
+        name: "Popup",
+    },
+    {
+        id: "s_media_list",
+        name: "Media List",
+    },
+    {
+        id: "s_three_columns",
+        name: "Columns",
+    },
+];
+
+const setOnScrollAnim = function () {
+    return [
+        wTourUtils.changeOption("WebsiteAnimate", 'we-select[data-is-animation-type-selection="true"] we-toggler'),
+        wTourUtils.changeOption("WebsiteAnimate", 'we-button[data-animation-mode="onScroll"]'),
+        wTourUtils.changeOption("WebsiteAnimate", 'we-select[data-name="animation_effect_opt"] we-toggler'),
+        wTourUtils.changeOption("WebsiteAnimate", 'we-button[data-name="o_anim_slide_in_opt"]'),
+    ];
+};
+
+wTourUtils.registerWebsitePreviewTour("snippet_popup_and_animations", {
+    test: true,
+    url: "/",
+    edition: true,
+}, [
+    wTourUtils.dragNDrop(snippets[1]), // Media List
+    wTourUtils.dragNDrop(snippets[1]), // Media List
+    wTourUtils.dragNDrop(snippets[2]), // Columns
+    wTourUtils.clickOnElement("3rd columns", "iframe .s_three_columns .row > :last-child"),
+    ...setOnScrollAnim(),
+    {
+        trigger: ".o_we_invisible_el_panel .o_we_invisible_entry",
+        content: "Open the Cookies Bar.",
+        run: "click",
+    },
+    {
+        trigger: "iframe #wrapwrap",
+        extra_trigger: "iframe #website_cookies_bar:not(.d-none)",
+        content: "Scroll to top",
```

Comment:
```suggestion
        content: "Scroll to top",
        trigger: "iframe #wrapwrap",
        extra_trigger: "iframe #website_cookies_bar:not(.d-none)",  
```
Nitpicking but not so much: having `content` first is just a nice to have, whenever you read a test to figure what it does (because you need to review or debug it), you only care about this part to have an idea of what it does, like reading steps of a manual.
If it's lost in the middle of the step's properties, the tour becomes impossible to read easily.
On top of that, having the `trigger` first doesn't really bring any value that would make it so we want it first.